corehq/apps/locations/models.py

```diff
@@ -438,7 +438,8 @@ def __init__(self, *args, **kwargs):
             self.location_type = location_type
 
     def __repr__(self):
-        return "%s (%s)" % (self.name, self.location_type)
+        val = u"%s (%s)" % (self.name, self.location_type)
+        return val.encode('utf-8')
```

Comment:
@esoergel I'm not sure, I haven't really worked on much supply stuff, but the place where it was being called from to cause the issue that this PR fixed was here: https://github.com/dimagi/commcare-hq/blob/0fcff600792ef1a7239a890f8c7f15617d23e0e7/corehq/apps/commtrack/sms.py#L68